src/actions/update.rs

```diff
@@ -2,34 +2,12 @@
 
 use std::path::PathBuf;
 
-use crate::{cli::UpdateCommand, command, log, paths};
+use crate::{cli::UpdateCommand, command, log};
 
 /// Runs `cargo update` on project and backends in .builder* folders.
```

Comment:
The doc is outdated, there are no backends anymore. Very likely there are more places to edit.